src/mongodbTypes.ts

```diff
@@ -0,0 +1,185 @@
+import type {
```

Comment:
if re-ordering the functions in this file is not preferred, does it make sense to disable that lint rule on this entire file?